halo2_gadgets/src/ecc/chip/mul_fixed/full_width.rs

```diff
@@ -249,8 +258,12 @@ pub mod tests {
                         .fold(pallas::Scalar::zero(), |acc, c| {
                             acc * &h + &pallas::Scalar::from(c.to_digit(8).unwrap() as u64)
                         });
-            let (result, _) =
-                base.mul(layouter.namespace(|| "mul with double"), Some(scalar_fixed))?;
+            let by = ScalarFixed::new(
+                chip.clone(),
+                layouter.namespace(|| "1333333333333333333333333333333333333333333333333333333333333333333333333333333333334"),
```

Comment:
not blocking: We could put this string (which also appears above) into a `&'static str`.